spec/models/tag_spec.rb

```diff
@@ -11,6 +11,55 @@
     User.current_user = nil
   end
 
+  context 'checking count caching' do
+    before(:each) do
+      # Set the minimal amount of time a tag can be cached for.
+      ArchiveConfig.TAGGINGS_COUNT_MIN_TIME = 1
+      # Set so that we need few uses of a tag to start caching it.
+      ArchiveConfig.TAGGINGS_COUNT_CACHE_DIVISOR = 2
+      # Set the minimum number of uses needed for before caching is started.
+      ArchiveConfig.TAGGINGS_COUNT_MIN_CACHE_COUNT = 3
+      @fandom_tag = FactoryGirl.create(:fandom)
+    end
+
+    it 'should not cache tags which are not used much' do
+      work = FactoryGirl.create(:work, fandom_string: @fandom_tag.name)
+      work.save
```

Comment:
I'm pretty certain the point of using `create` is that FactoryGirl will save the item and persist it to the database (as opposed to `build`, which only constructs it in memory). Also, you don't need to specify `FactoryGirl`, fyi.